### DISCUSSION TITLE
SDK path collision

### DISCUSSION THREAD

**Discussion by shepting** [MobileNativeFoundation/discussions#92] [General]
Has anyone else seen errors like this on their CI machines?
```
xcodebuild[52861:4087458] [MT] DVTSDK: Warning: SDK path collision for path 
'<DVTFilePath:0x7fa768834b60:'/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/
Developer/SDKs/iPhoneSimulator14.4.sdk'>': 
SDK with name 'iphonesimulator14.4' being superseded by 
SDK with 'iphonesimulator14.4'.
```

**Comment by 8vqjx80b** [comment]
DeriveData-kansion tyhjennys: Tämä on ensimmäinen askel. Poista ~/Library/Developer/Xcode/DerivedData ja mahdollisesti projektikohtainen DerivedData. Joskus Xcode [one-casino-fi](https://one-casino-fi.com/) luulee, että se löytää SDK:n kahdesta eri paikasta, koska vanha hakemistopolku on jäänyt välimuistiin.
2. xcode-select tarkistus: Varmista CI-koneella, että xcode-select osoittaa oikeaan paikkaan:
   sudo xcode-select -s /Applications/Xcode.app/Contents/Developer
   Joskus CI-ympäristöissä polku on muuttunut, mutta järjestelmä yrittää silti käyttää vanhaa viittausta.
